sdk/src/types/block/payload/signed_transaction/transaction.rs

```diff
@@ -324,6 +324,15 @@ impl Transaction {
     }
 }
 
+impl WorkScore for Transaction {
+    fn work_score(&self, params: WorkScoreParameters) -> u32 {
+        self.inputs().work_score(params)
+            + self.context_inputs().work_score(params)
+            + self.outputs().work_score(params)
+            + self.mana_allotments().work_score(params)
```

Comment:
Bit nitty but can you order the same as the fields pls

Reply:
Done